packages/agents/sequencer/src/lib/operations/execute/fastpath.ts

```diff
@@ -52,6 +52,13 @@ export const storeFastPathData = async (bid: Bid, _requestContext: RequestContex
     });
   }
 
+  // Update the last bid time for a given router.
+  await cache.routers.setLastBidTime(bid.router, {
+    originDomain: transfer.xparams.originDomain,
```

Comment:
Would also be useful for routers to know when the last successful/accepted bid was.
Wdyt ?
@wanglonghong

Reply:
That's good suggestion.